Build.proj

```diff
@@ -0,0 +1,72 @@
+<Project Sdk="Microsoft.Build.Traversal">
+
+  <!--
+    Subsets are already imported by Directory.Build.props.
```

Comment:
So if we're building just one subsetCategory, only that item will be set and the others will be empty I guess?

Reply:
correct, the other items will be empty.